arangod/Cluster/Maintenance.cpp

```diff
@@ -84,6 +85,13 @@ static VPackValue const VP_SET("set");
 static std::string_view const PRIMARY("primary");
 static std::string_view const EDGE("edge");
 
+namespace {
+bool isViewIndex(std::string_view indexType) noexcept {
+  return indexType == iresearch::StaticStrings::ViewArangoSearchType ||
```

Comment:
Only indexType == arangosearch_link_type needed

Reply:
Thanks will fix this :+1: